src/Makefile

```diff
@@ -1,5 +1,5 @@
 INCLUDE_DIRS = ../inc
-CFLAGS += -O2
+CFLAGS += -O2 -fPIC
```

Comment:
I believe this to be what we want all the time

https://stackoverflow.com/questions/19364969/compilation-fails-with-relocation-r-x86-64-32-against-rodata-str1-8-can-not